public/js/user.js

```diff
@@ -0,0 +1,56 @@
+apos.define('apostrophe-login-hcaptcha', {
+  construct: function(self, options) {
+    var $form = $('[data-apos-login-form]');
```

Comment:
I know this is A2 so we are stuck with `var` here, but please figure out why github actions is warning about it inline here and correct that for the `public` folder.